src/components/AboutPage.astro

```diff
@@ -12,17 +12,17 @@
       <a href="https://twitter.com/t3dotgg" class="link-dark">Theo</a>,
       <a href="https://mark.florkow.ski" class="link-dark">Mark</a>
       and
-      <a href="https://github.com/JasonDocton" class="link-dark">Jason</a>
+      <a href="https://github.com/JasonDocton" class="link-dark">Jason</a>.
     </p>
     <p>
       Hosted on
-      <a href="https://vercel.com" class="link-dark"> Vercel </a>
+      <a href="https://vercel.com" class="link-dark">Vercel</a>.
     </p>
     <p>
       Source available on
       <a href="https://github.com/TheoBr/init.tips" class="link-dark">
-        Github
-      </a>
+        Github</a
+      >.
```

Comment:
@roj1512 What is the purpose/point of separating tags onto two lines?

Reply:
Astro’s formatter did that.

Reply:
Thank you for your answer @roj1512